application/electron/src/services/service.sessions.ts

```diff
@@ -0,0 +1,55 @@
+import Logger from '../tools/env.logger';
+import { IService } from '../interfaces/interface.service';
+import ServiceElectron, { IPCMessages, Subscription } from './service.electron';
+
+/**
+ * @class ServiceSessions
+ * @description Just keep information about sessions
+ */
+
+class ServiceSessions implements IService {
```

Comment:
I think we don't need one more service, just for one additional IPC message. Even more, I don't think we need additional IPC (SessionChange). You can make little modification in:
```application/electron/src/services/service.streams.ts```
Like this:
```
    private _ipc_onStreamSetActive(message: IPCElectronMessages.TMessage) {
        if (!(message instanceof IPCElectronMessages.StreamSetActive)) {
            return;
        }
        if (this._activeStreamGuid === message.guid) {
            return;
        }
        if (this._streams.has(message.guid)) {
            // This is real session
            this._activeStreamGuid = message.guid;
            this._subjects.onSessionChanged.emit(message.guid);
            this._logger.debug(`Active session is set to: ${this._activeStreamGuid}`);
        } else {
            // This is custom tab (like About, Plugins, Settings etc.)
            this._logger.debug(
                `Active session would not set to: ${message.guid} as soon it isn't stream realated session`,
            );
        }
        ServiceElectron.updateMenu();
    }
```